test/e2e-go/cli/goal/expect/tealdbgTest.exp

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/expect -f
+set err 0
+log_user 1
+
+if { [catch {
+    source  goalExpectCommon.exp
+
```

Comment:
Suggest increasing timeout from the default of 30 seconds to 60 seconds.

    `set timeout 60`

Reply:
done